scripts/lib.star

```diff
@@ -71,7 +71,7 @@ def slack_step(channel):
                 'from_secret': 'slack_webhook',
             },
             'channel': channel,
-            'template': 'Build {{build.number}} failed: {{build.link}}',
+            'template': 'Build {{build.number}} failed: {{build.link}} for PR: {{build.pull}} \n Author: {{build.author}}',
```

Comment:
I think it'll be more readable with only one colon per sentence, did you intentionally add space after the newline for indentation? Otherwise, I think it's cleaner to remove the extra space.
```suggestion
            'template': 'Build {{build.number}} failed for PR {{build.pull}}: {{build.link}}\nAuthor: {{build.author}}',
```

Reply:
Yeah, I did it on purpose just to be more distinguishable, but I think you are right, the less lines of slack posts the better!
However, it may need more than 1 line. Fixing... :)

Reply:
@dsotirakis I wasn't saying to change the number of lines, there're still two lines after my edit right?

Reply:
I just removed some whitespace before and after the newline (the whitespace before the newline should be invisible, and the whitespace after lead to a one-space indent).

Reply:
Oops, totally missed the suggestion, misunderstood! Changing